view/templates/item/compose.tpl

```diff
@@ -97,7 +97,6 @@
 </div>
 <script>
 	Dropzone.autoDiscover = false;
-	console.log('compose.tpl', {{$max_imagesize}} / 100000);
 	var maxis = {{$max_imagesize}} / 100000;
```

Comment:
This doesn't look correct, the divisor probably should be `1000000` (10^6). This said, I would prefer if this calculation was performed when the template value is set rather than here.